react/global.css

```diff
@@ -1,3 +1,47 @@
-.vtex-impersonate-customer {
+.vtex-telemarketing {
+  text-align: center;
+}
+
+.vtex-telemarketing .vtex-popover__arrow-up {
```

Comment:
i think we have the colors in vtex-tachyons https://vtex.github.io/vtex-tachyons/

Reply:
we do, however, in this case, I need to override the color, that's why I can´t use tachyons.